### PR TITLE
Schedule submission times individually bugs

### PR DESCRIPTION
**Pull request by naazy**
able to schedule submission times in programme builder using IE11/ Edge https://github.com/OxfordAbstracts/application/issues/3791

9:60 no longer appears as a possible time for a submission in a session https://github.com/OxfordAbstracts/application/issues/3804

### CODE REVIEW

**Comment by naazy** [comment]
@roryc89 you will need to diff against https://github.com/OxfordAbstracts/elm-program-builder/pull/99 :pray:

File: src/Helpers.elm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module Helpers exposing (onChange)
```
Comment (roryc89) [inline src/Helpers.elm]: 👍 for a useful abstraction

File: src/Helpers.elm [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+import Html exposing (..)
```
Comment (roryc89) [inline src/Helpers.elm]: Do we need to import both `Html` and `Html.Attribute`? I assume one of the provides `Attribute`

Comment (naazy) [inline src/Helpers.elm]: Have removed Html.Attribute

**Pull request merged**